tests/acceptance/features/apiContract/propfind.feature

```diff
@@ -46,3 +46,54 @@ Feature: Propfind test
       | manager | RDNVCKZP      |
       | editor  | DNVCK         |
       | viewer  |               |
+
+
+  Scenario Outline: space member with a different role checks the PROPFIND request of folders in the space
+    Given user "Alice" has created a folder "folderMain/subFolder1/subFolder2" in space "new-space"
+    And user "Alice" has shared a space "new-space" with settings:
+      | shareWith | Brian  |
+      | role      | <role> |
+    When user "Brian" sends PROPFIND request from the space "new-space" to the resource "folderMain" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response should contain a space "new-space" with these key and value pairs:
+      | key            | value             |
+      | oc:fileid      | UUIDof:folderMain |
+      | oc:file-parent | UUIDof:new-space  |
+      | oc:name        | folderMain        |
+      | oc:permissions | <oc_permission>   |
+      | oc:size        | 0                 |
+    When user "Brian" sends PROPFIND request from the space "new-space" to the resource "folderMain/subFolder1/subFolder2" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response should contain a space "new-space" with these key and value pairs:
+      | key            | value                                   |
+      | oc:fileid      | UUIDof:folderMain/subFolder1/subFolder2 |
+      | oc:file-parent | UUIDof:folderMain/subFolder1            |
+      | oc:name        | subFolder2                              |
+      | oc:permissions | <oc_permission>                         |
+      | oc:size        | 0                                       |
+    Examples:
+      | role    | oc_permission |
+      | manager | RDNVCKZP      |
+      | editor  | DNVCK         |
+      | viewer  |               |
```

Comment:
how about splitting this into 2 scenarios like `root-folder` and for `sub-folder`. Scenario description can differentiate in a good way.